apps/etterlatte-opplysninger-fra-soeknad/src/test/kotlin/no/nav/etterlatte/opplysningerfrasoknad/OpplysningsuthenterBarnepensjonTest.kt

```diff
@@ -122,7 +122,8 @@ internal class OpplysningsuthenterBarnepensjonTest {
                 assertNull(iban)
                 assertNull(swift)
                 assertNull(oenskerSkattetrekk)
-                assertNull(oensketSkattetrekkProsent)
+                assertNull(oensketSkattetrekk)
```

Comment:
Samme to ganger?

Reply:
Er nok ikke så synlig forskjell, men den første er `oenskeR` og den andre `oenskeT`